pkgs/applications/misc/ulauncher/default.nix

```
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, python27Packages,
+  gnome3, gobject-introspection, wrapGAppsHook, gtk3, webkitgtk,
+  libnotify, keybinder3, libappindicator, intltool, wmctrl }:
+
+python27Packages.buildPythonApplication rec  {
+  pname = "ulauncher";
+
+  versionNumber = "4.3.2";
+  version = "${versionNumber}.r8";
+
+  src = fetchurl {
+    url = "https://github.com/Ulauncher/Ulauncher/releases/download/${version}/ulauncher_${version}.tar.gz";
+    sha256 = "0494326s84nvyhblcghxzidp2qf0v0l61kyrf8a92qqryqnzmwip";
+  };
+
+  nativeBuildInputs = with python27Packages;  [
+    distutils_extra
+    intltool
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    keybinder3
+    libappindicator
+    libnotify
+    gnome3.adwaita-icon-theme
+    gobject-introspection
+    webkitgtk
+    wmctrl
+  ];
+
+  propagatedBuildInputs = with python27Packages; [
+    dbus-python
+    notify
+    pygobject3
+    pyinotify
+    pysqlite
+    python-Levenshtein
+    pyxdg
+    websocket_client
+  ];
+
+  doCheck = false;
+  checkInputs = with python27Packages; [ mock ];
+
+  patchPhase = ''
+    substituteInPlace setup.py --replace "target_data + 'share/ulauncher/'" "'$out/share/ulauncher/'"
+    substituteInPlace ulauncher/config.py --replace "VERSION" "${version}"
+  '';
+
+  preFixup = ''
+  	gappsWrapperArgs+=(--prefix PATH : "${stdenv.lib.makeBinPath [ wmctrl ]}")
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A fast application launcher for Linux, written in Python, using GTK+";
```

```suggestion
    description = "A fast application launcher for Linux, written in Python, using GTK";
```

GTK finally dropped the `+`.

---

Yay, we are probably full of this

---

opened https://github.com/NixOS/nixpkgs/issues/57887